base/sysinfo.jl

```diff
@@ -165,6 +165,18 @@ mutable struct UV_cpu_info_t
     cpu_times!idle::UInt64
     cpu_times!irq::UInt64
 end
+"""
```

Comment:
blank line before docstring:
```suggestion

"""
```